var/spack/repos/builtin/packages/vtk/package.py

```diff
@@ -40,10 +41,18 @@ class Vtk(CMakePackage):
     # At the moment, we cannot build with both osmesa and qt, but as of
     # VTK 8.1, that should change
     conflicts('+osmesa', when='+qt')
+    conflicts('+python', when='+python3')
+    conflicts('+python3', when='@:7')
```

Comment:
Sorry to keep on this but after you clarified, your original spec of `@:8.0` was good and makes more sense to do in this case so that it's "current versions and on"

Reply:
No, problem, I can change it back.